### PR TITLE
Update link for delete supplier in UserGuide

### PR DESCRIPTION
**Pull request by prave1n**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Vendor Vault is a **desktop app for managing supplier contact information and de
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
 **[Add](#adding-a-supplier-add-s)**    | `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​` <br> e.g., `add -s n/John Doe p/98765432 e/johnd@example.com com/companyA t/friends t/owesMoney pro/rice pro/bread`
-**[Delete](#deleting-a-supplier--delete-s)** | `delete -s INDEX`<br> e.g., `delete -s 3`
+**[Delete](#deleting-a-supplier-delete-s)** | `delete -s INDEX`<br> e.g., `delete -s 3`
```
Comment (vinc3leong) [inline docs/UserGuide.md:70]: LGTM

**Review by vinc3leong — COMMENTED** [review]
LGTM

**Review by CJianzhi — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -77,19 +77,23 @@ Action     | Format, Examples
...
 
 
 
 ### General Commands
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**List**   | `list -a`
-**Help**   | `help`
-**Exit**   | `exit`
+**[List](#viewing-all-deliveries-and-suppliers-list)**   | `list`
```
Comment (ngjianyi) [inline docs/UserGuide.md:93]: Should this be `list -a` instead?

**Review by ngjianyi — COMMENTED** [review]
LGTM

**Pull request merged**